docs/networks/mainnet/running-a-node.md

```diff
@@ -243,14 +243,25 @@ We also created a "docker factory" web interface which allows you to generate yo
 
 Difficulty: Advanced 🌶️🌶️🌶️🌶️
 
-You are in no way limited to the tools we provide. If you are a pro user, you can download and run the Erigon, Geth, Lighthouse, Prysm or any other clients in your preferred setup. Please refer to the LUKSO [network configuration](https://github.com/lukso-network/network-configs/) repository to configure and start your node.
+You are in no way limited to the tools we provide. If you are a pro user, you can download and run the Erigon, Geth, Lighthouse, Prysm or any other clients in your preferred setup. Please refer to the LUKSO Mainnet [network configuration](https://github.com/lukso-network/network-configs/tree/main/mainnet) repository to configure and start your node.
 
-Additionaly, some client support LUKSO out of the box - this means that you don't need to prepare your chain configs beforehand and simply use all the tools provided by that client. List of said clients (with their respective guides) includes:
+Additionally, some clients support LUKSO out of the box. This means that you don't need to prepare your chain configurations beforehand and can simply use all the tools provided by that client. The list of these clients (with their respective guides) includes:
 
-- [Besu](https://besu.hyperledger.org/public-networks/reference/cli/options#network)
-- [Teku](https://docs.teku.consensys.io/reference/cli#network)
+Execution layer clients:
 
-:::info LUKSO Network configuration
+- [Besu](https://besu.hyperledger.org/public-networks/reference/cli/options#network) Example command: `besu --network=lukso` (basic network flag, you should consider adding more flags, e.g. `--data-path=<PATH>`)
```

Comment:
Suggest putting the command as whole code block, instead of code text. So that user can copy the command by clicking on the top right icon.

---

- [Besu](https://besu.hyperledger.org/public-networks/reference/cli/options#network) Example command (basic network flag, you should consider adding more flags, e.g. `--data-path=<PATH>`)

```bash
besu --network=lukso
```

Reply:
Done